File: FilFeed/src/main/java/com/phicdy/mycuration/util/TextUtil.java
package com.phicdy.mycuration.util;

import java.util.regex.Pattern;

public class TextUtil {

    public static String removeLineFeed(String string) {
        return Pattern.compile("\t|\r|\n|\r\n").matcher(string).replaceAll("");
    }

    public static boolean isEmpty(String text) {
        if (text == null || text.equals("")) {
            return true;
        }
        return false;
    }
}

File: legacy/src/main/java/com/phicdy/mycuration/util/TextUtil.kt
package com.phicdy.mycuration.util

import java.util.regex.Pattern

object TextUtil {

    fun removeLineFeed(string: String): String {
        return Pattern.compile("\t|\r|\n|\r\n").matcher(string).replaceAll("")
    }

    fun isEmpty(text: String?): Boolean {
        return text == null || text == ""
    }
}
